qadence/backends/pulser/config.py

```diff
@@ -81,6 +82,9 @@ class Configuration(BackendConfiguration):
     # configuration for cloud simulations
     cloud_configuration: Optional[CloudConfiguration] = None
 
+    addressing_pattern: AddressingPattern | None = None
+    """Semi-local addressing pattern."""
+
```

Comment:
How do you specify an addressing pattern for the PyQ backend?

Reply:
Currently it's passed as an argument to `add_interaction` function. This of course will change with rework of analog interface.